applications/conversations/views/messages/conversations.php

```diff
@@ -32,7 +32,7 @@
     $CssClass .= ' '.($Conversation->CountNewMessages <= 0 ? 'Read' : 'Unread');
 
     $JumpToItem = $Conversation->CountMessages - $Conversation->CountNewMessages;
-    $Message = (sliceString(Gdn_Format::plainText($Conversation->LastBody, $Conversation->LastFormat), 100));
+    $Message = (sliceString(Gdn::formatService()->renderExcerpt($Conversation->LastBody, $Conversation->LastFormat), 100));
```

Comment:
First `htmlspecialchars` was part of `Gdn_Format::plainText`.

Second one is here on the output line: https://github.com/vanilla/vanilla/blob/97472338dc7ca97e044eb2fea7c068eb318fc74c/applications/conversations/views/messages/conversations.php#L69

`renderPlainText()` from the format service does not apply this html escaping.